conans/test/integration/package_id/test_cache_compatibles.py

```diff
@@ -0,0 +1,30 @@
+import os
+import re
+import textwrap
+
+from conans.test.utils.tools import TestClient, GenConanfile
+from conans.util.files import save
+
+
+class TestCacheCompatibles:
+
+    def test_compatible_build_type(self):
+        client = TestClient()
+        compatibles = textwrap.dedent("""\
+            def compatibility(conanfile):
+                result = []
```

Comment:
- Make sure that we can split the file in several and import them
- Make sure that we can use the recipe reference as data to compatibility
- Move it to extensions folder

Reply:
Done the 3 things.

Regarding the extensions folder it is in ``extensions/plugins/binary_compatibility.py``